### ISSUE TITLE
Problems with Channels

### ISSUE THREAD

**Issue by mferioli** [Dalmirog/OctoPosh#150]
I'm not sure if the issue is on Projects that use Channels or where a Project has Variables scoped to a Channel, but I get this response on Get-OctopusVariableSet:

```
PS C:\Users\mferioli> Get-OctopusVariableSet -Projectname "MyProject"
Exception calling "Get" with "1" argument(s): "Could not convert string 'Channel' to dictionary key type 'Octopus.Platform.Model.ScopeField'. Create a TypeConverter to convert from the st
At C:\Program Files\WindowsPowerShell\Modules\Octoposh\0.2.72\scripts\Get-OctopusVariableSet.ps1:98 char:13
+             $vs = $c.repository.VariableSets.Get($id)
+             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [], MethodInvocationException
    + FullyQualifiedErrorId : JsonSerializationException
```

**Comment by Dalmirog** [comment]
Hi! This is because the latest version of Octoposh is using the Octopus client library from `2.6`, and channels didn't exist in that version. 

This will be fixed by #148 . I'm gonna try to ship a build with that fix by the end of today.

**Comment by mferioli** [comment]
Worked like a charm.  Thanks and keep up the good work.  Love this module.
